setup.py

```diff
@@ -63,43 +66,92 @@
 
     'Sphinx',
 
-    # For the bodhi-client
-    'click',
-
     'WebOb>=1.4.1',
     ]
 
 if sys.version_info[:3] < (2,7,0):
-    requires.append('importlib')
+    server_requires.append('importlib')
 
 if sys.version_info[:3] < (2,5,0):
-    requires.append('pysqlite')
+    server_requires.append('pysqlite')
+
+
+setuptools.command.egg_info.manifest_maker.template = 'BODHI_MANIFEST.in'
+
 
 setup(name='bodhi',
-      version='2.1.9',
-      description='bodhi',
+      version=VERSION,
+      description='bodhi common package',
       long_description=README + '\n\n' +  CHANGES,
       classifiers=[
         "Programming Language :: Python",
-        "Framework :: Pyramid",
-        "Topic :: Internet :: WWW/HTTP",
-        "Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
         ],
       author='',
       author_email='',
       url='',
```

Comment:
Yeah we could do these, but I'd prefer to do them in a separate PR since they were already like this before. I'll add a task to my task warrior to do this.